spec/features/bucket_lists/user_deletes_bucket_list_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+feature 'authenticated user deletes a bucket list', %(
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.